src/components/Presets/Presets.js

```diff
@@ -25,17 +32,16 @@ function mix (a, b) {
 }
 
 const presets = {
-  'None': null,
-  'Defaults': null,
+  'Defaults': {},
   'Mining': mining,
-  'Non-standard Ports': ports
-  // 'Low Bandwidth': bandwidth
-  // Pruned Node
-  // Raspberry Pi https://github.com/MrChrisJ/fullnode/blob/master/Setup_Guides/bitcoin.conf
-  // Tor Node     https://gist.github.com/MrChrisJ/1098de03ef9588943fc7
-  // Testnet node
-  // Regtest node
-  // Non-syncing node
+  'Non-standard Ports': ports,
+  'Low Bandwidth': bandwidth,
+  'Low Disk Usage': pruned,
+  'Raspberry Pi': pi,
+  'Privacy': tor,
+  'Testnet': testnet,
+  'Regtest': regtest,
+  'Non-syncing': nosync
```

Comment:
we need to add preset configs for nosync, right now it's empty.

I think all it needs is `connect = 0`